src/openzaak/conf/includes/base.py

```diff
@@ -775,6 +776,10 @@
     "ZAAK_EIGENSCHAP_WAARDE_VALIDATION", default=False
 )
 
+# Import settings
+IMPORT_DOCUMENTEN_BASE_DIR = config("IMPORT_DOCUMENTEN_BASE_DIR", BASE_DIR)
```

Comment:
The whole process should be documented including these env vars should be documented. Please don't forget that nice sequence diagram, which is very useful. It can be done in the separate PR though (but it's necessary before we release it)

Reply:
Discussed today, will be added through a separate PR